pytesmo/metrics.py

```diff
@@ -198,53 +270,84 @@ def pearson_conf(r, n, c=95):
     return np.tanh(z_lower), np.tanh(z_upper)
 
 
-def spearmanr(x, y):
+def spearmanr(o, p):
     """
-    Wrapper for scipy.stats.spearmanr
+    Wrapper for scipy.stats.spearmanr. Calculates a Spearman
+    rank-order correlation coefficient and the p-value to
+    test for non-correlation.
 
     Parameters
     ----------
-    x : numpy.array
+    x: numpy.array
         1D numpy array to calculate the metric
-    y : numpy.array
+    y: numpy.array
         1D numpy array to calculate the metric
 
     Returns
     -------
-    rho : float
+    rho: float
         Spearman correlation coefficient
-    p-value : float
-        The two-sided p-value for a hypothesis test whose null hypothesis
+    p - value: float
```

Comment:
Why the spaces between p-value and two-sided? They are both written without. e.g.: [p-value](http://en.wikipedia.org/wiki/P-value)